specs/electra/beacon-chain.md

```diff
@@ -1431,10 +1470,6 @@ def process_consolidation_request(
     source_validator = state.validators[source_index]
     target_validator = state.validators[target_index]
 
-    # Verify that source != target, so a consolidation cannot be used as an exit.
-    if source_index == target_index:
-        return
-
     # Verify source withdrawal credentials
     has_correct_credential = has_execution_withdrawal_credential(source_validator)
     is_correct_source_address = (
```

Comment:
lets do this later but I think we can also factor out a helper like `is_valid_consolidation_request`